### PR TITLE
Refact worker

### PR DESCRIPTION
**Pull request by MaianaB**
**Feature/Bug description:** The file "worker" has several handlers and auxiliary functions.

**Solution:** The file was refactored so that the handlers responsible for permissions-related changes were moved to another file, as well as to the helper functions. These functions have been moved to the utils worker file.

**TODO/FIXME:**Refactor the remaining handlers in the file. The worker should only contain the routes.

### CODE REVIEW

File: backend/test/institution_handlers_tests/institution_handler_test.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 from models import User
 from models import Institution
 from handlers.institution_handler import InstitutionHandler
-from worker import AddAdminPermissionsInInstitutionHierarchy
-from worker import RemoveAdminPermissionsInInstitutionHierarchy
+from worker_handler import AddAdminPermissionsInInstitutionHierarchyHandler
```
Comment (raonismaneoto) [inline backend/test/institution_handlers_tests/institution_handler_test.py]: You could make these 2 imports in the same line separating them by comma

File: backend/test/institution_handlers_tests/institution_hierarchy_handler_test.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 from ..test_base_handler import TestBaseHandler
 from handlers.institution_hierarchy_handler import InstitutionHierarchyHandler
-from worker import AddAdminPermissionsInInstitutionHierarchy
-from worker import RemoveAdminPermissionsInInstitutionHierarchy
+from worker_handler import AddAdminPermissionsInInstitutionHierarchyHandler
```
Comment (raonismaneoto) [inline backend/test/institution_handlers_tests/institution_hierarchy_handler_test.py]: same thing

File: backend/worker.py [hunk 3]
```diff
@@ -121,15 +37,14 @@ def remove_permissions(remove_hierarchy, institution):
             for child in institution.children_institutions:
                 child = child.get()
                 if(child.verify_connection(institution, 'PARENT')):
-                    remove_permissions(remove_hierarchy, child)
+                    remove_hierarchy_permissions(remove_hierarchy, child)
```
Comment (raonismaneoto) [inline backend/worker.py]: I didn't get it. You've renamed the method?

File: backend/test/institution_handlers_tests/institution_handler_test.py [hunk 4]
```diff
@@ -6,8 +6,8 @@
...
+from worker_handler import RemoveAdminPermissionsInInstitutionHierarchyHandler
```
Comment (ruanres) [inline backend/test/institution_handlers_tests/institution_handler_test.py]: Simplify those handler's names. I think you can remove the 'InInstitutionHierarchy' part.

File: backend/worker_handler/utils_worker.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
+
```
Comment (raonismaneoto) [inline backend/worker_handler/utils_worker.py]: I think you could rename this file to worker_aux_functions, as we are using that, or even change the order, worker_utils makes more sense.

File: backend/test/institution_handlers_tests/institution_hierarchy_handler_test.py (hunk 2 above)
Comment (raonismaneoto) [inline backend/test/institution_handlers_tests/institution_hierarchy_handler_test.py]: It should call worker_handlers*

File: backend/worker_handler/permissions/transfer_admin_permissions_handler.py [hunk 6]
```diff
@@ -0,0 +1,102 @@
+from google.appengine.ext import ndb
+from util import NotificationsQueueManager
+from permissions import DEFAULT_SUPER_USER_PERMISSIONS
+from worker_handler import filter_permissions_to_remove
```
Comment (raonismaneoto) [inline backend/worker_handler/permissions/transfer_admin_permissions_handler.py]: simplify the imports please

File: backend/worker_handler/permissions/admin_permissions_inst_hierarchy_handler.py [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+from worker_handler import is_not_admin
+from worker_handler import should_remove
+from worker_handler import BaseHandler
+from google.appengine.ext import ndb
+from util import NotificationsQueueManager
+
+
+__all__ = ['RemoveAdminPermissionsInInstitutionHierarchyHandler', 'AddAdminPermissionsInInstitutionHierarchyHandler']
+
+class RemoveAdminPermissionsInInstitutionHierarchyHandler(BaseHandler):
```
Comment (ruanres) [inline backend/worker_handler/permissions/admin_permissions_inst_hierarchy_handler.py]: I think a better name for this file would be 'manage_admin_permission_handler.py'

File: backend/worker.py (hunk 3 above)
Comment (MaianaB) [reply backend/worker.py]: Sim, pq antes ele tinha outra função com o mesmo nome dentro do arquivo. Essa função no caso remove as permissões hierarquicamente também.

File: backend/test/institution_handlers_tests/institution_handler_test.py (hunk 1 above)
Comment (MaianaB) [reply backend/test/institution_handlers_tests/institution_handler_test.py]: Acredito que fica muito grande.

**Pull request closed without merging**